### PR TITLE
Add docs for push notification integrations

### PR DESCRIPTION
**Pull request by hebime**

### CODE REVIEW

File: slate/source/includes/rest/_managed.md [hunk 1]
```diff
@@ -631,6 +631,89 @@ To configure an Email integration, simply call the Create Integration endpoint w
...
+```
+201 CREATED
+```
+```json
+{
+  "integration": {
+    "type": "apn",
+    "_id": "58878a8842fadcdb7b70b74c",
+    "production": false,
+	"autoUpdateBadge": false
```
Comment (mspensieri) [inline slate/source/includes/rest/_managed.md]: indentation weirdness here

File: slate/source/includes/rest/_managed.md [hunk 2]
```diff
@@ -631,6 +631,89 @@ To configure an Email integration, simply call the Create Integration endpoint w
...
+
+```
+201 CREATED
+```
+```json
+{
+  "integration": {
+    "_id": "5876a3d4abf286d0c0af1467",
+    "type": "fcm",
+    "senderId": 1429457686312
```
Comment (mspensieri) [inline slate/source/includes/rest/_managed.md]: Isn't this a string now?

File: slate/source/includes/rest/_managed.md [hunk 3]
```diff
@@ -631,6 +631,89 @@ To configure an Email integration, simply call the Create Integration endpoint w
...
+```
+
+To configure a Firebase Cloud Messaging integration, first visit the [Firebase Console](https://console.firebase.google.com/).
+Copy the `serverKey` and `senderId` from the `Cloud Messaging` tab in the settings of your project and call the create
+integrations endpoint with this data.
+
+| **Arguments**                                      |                                       |
+|----------------------------------------------------|---------------------------------------|
+| **type**<br/><span class='req'>required</span>     | The integration type: _fcm_.          |
+| **severKey**<br/><span class='req'>required</span> | Your server key from the fcm console. |
```
Comment (mspensieri) [inline slate/source/includes/rest/_managed.md]: typo: severkey

File: slate/source/includes/rest/_managed.md [hunk 4]
```diff
@@ -631,6 +631,89 @@ To configure an Email integration, simply call the Create Integration endpoint w
...
+| **senderId**<br/><span class='req'>required</span> | Your sender id form the fcm console.  |
```
Comment (mspensieri) [inline slate/source/includes/rest/_managed.md]: typo: form

File: slate/source/includes/rest/_managed.md [hunk 5]
```diff
@@ -631,6 +631,89 @@ To configure an Email integration, simply call the Create Integration endpoint w
...
+```shell
+curl https://api.smooch.io/v1/apps/55c8d9758590aa1900b9b9f6/integrations \
+     -X POST \
+     -d '{"type": "apn", "certificate": "HjkUD4rWvZj7wSDzA8Hu2hd7ICs274Z=="}' \
+     -H 'content-type: application/json' \
+     -H 'authorization: Bearer your-account-token'
+```
+
+```javascript
+// These endpoints are not currently wrapped in a JavaScript lib
```
Comment (jpjoyal) [inline slate/source/includes/rest/_managed.md]: Can you file this as a todo

File: slate/source/includes/rest/_managed.md [hunk 6]
```diff
@@ -631,6 +631,89 @@ To configure an Email integration, simply call the Create Integration endpoint w
...
+  "integration": {
+    "_id": "5876a3d4abf286d0c0af1467",
+    "type": "fcm",
+    "senderId": 1429457686312
+  }
+}
+```
+
+To configure a Firebase Cloud Messaging integration, first visit the [Firebase Console](https://console.firebase.google.com/).
+Copy the `serverKey` and `senderId` from the `Cloud Messaging` tab in the settings of your project and call the create
```
Comment (jpjoyal) [inline slate/source/includes/rest/_managed.md]: Can we link them to the Google console ?

File: slate/source/includes/rest/_managed.md (hunk 4 above)
Comment (hebime) [inline slate/source/includes/rest/_managed.md]: 👍

File: slate/source/includes/rest/_managed.md (hunk 3 above)
Comment (hebime) [inline slate/source/includes/rest/_managed.md]: 👍

File: slate/source/includes/rest/_managed.md (hunk 2 above)
Comment (hebime) [inline slate/source/includes/rest/_managed.md]: oops! forgot to update this after that change.

File: slate/source/includes/rest/_managed.md (hunk 1 above)
Comment (hebime) [inline slate/source/includes/rest/_managed.md]: uhgg didn't look that way in my file. Should be fixed now.

File: slate/source/includes/rest/_managed.md (hunk 6 above)
Comment (hebime) [inline slate/source/includes/rest/_managed.md]: Added it as well.

File: slate/source/includes/rest/_managed.md (hunk 5 above)
Comment (hebime) [inline slate/source/includes/rest/_managed.md]: Done 😉

File: slate/source/includes/rest/_managed.md [hunk 7]
```diff
@@ -631,6 +631,96 @@ To configure an Email integration, simply call the Create Integration endpoint w
...
+  }
+}
+```
+
+To configure an Apple Push Notification integration, call the create integration endpoint with a **base64** encoded Apple Push Notification certificate
+from the [Apple Developer Portal](https://developer.apple.com/).
+
+On MacOS or Linux you can run the following command in your terminal to base64 encode your certificate.
+
+`openssl base64 -in YOUR_CERTIFICATE.p12 | tr -d '\n'`
```
Comment (lemieux) [inline slate/source/includes/rest/_managed.md]: Do we expect most of our users to have javascript backends?

Comment (lemieux) [inline slate/source/includes/rest/_managed.md]: I'm not saying it's a bad idea to have a snippet.. just wondering if javascript is relevant.

Comment (jpjoyal) [inline slate/source/includes/rest/_managed.md]: Javascript is more relevant than the command line in fact.

**Comment by hebime** [comment]
new snippet style:
screen shot 2017-01-30 at 8 00 20 pm

**Pull request merged**